deep-learning/_posts/2018-03-22-detectionDistillation.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: review
+title:  "Knowledg Distillation on Object Detection"
+tags:   "Deep Learning"
+author: Faezeh Amjadi
+pdf:    https://papers.nips.cc/paper/6676-learning-efficient-object-detection-models-with-knowledge-distillation.pdf
+cite:
+  authors: "Guobin Chen, Wongun Choi, Xiang Yu, Tony X. Han, Manmohan Krishna Chandraker"
+  title:   "Learning Efficient Object Detection Models with
+Knowledge Distillation"
+  venue:   "NIPS2017"
+---
+
+### Description
+	
+They proposed an end-to-end trainable framework for learning multi-class object detection through knowledge distillation. A teacher network (a pretrained network with more parameter and deeper) supervises and leads to better student performance which the knowledg is received from teacher is more informative. 
```

Comment:
parameters*
knowledge*
which the knowledge is received from teacher is more informative.  (phrasing? It's not clear)